### PR TITLE
Update Telemetry Services Instructions in API Guide

### PR DESCRIPTION
**Pull request by sunithasomanna**
Updated Telemetry Services Instructions in API Readme with the following API URIs and their details:

/redfish/v1/TelemetryService
/redfish/v1/TelemetryService/MetricDefinitions
/redfish/v1/TelemetryService/MetricDefinitions/{MetricDefinitionID}
/redfish/v1//TelemetryService/MetricReportDefinitions
/redfish/v1/TelemetryService/MetricReportDefinitions/{MetricReportDefinitionID}
/redfish/v1/TelemetryService/MetricReports
/redfish/v1/TelemetryService/MetricReports/{MetricReportID}
/redfish/v1/TelemetryService/Triggers
/redfish/v1/TelemetryService/Triggers/{TriggerID}

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -486,9 +486,17 @@ Resource Aggregator for ODIM supports the following Redfish APIs:
...
 
-
-
-
+| TelemetryService                                             |                |
+| ------------------------------------------------------------ | -------------- |
+| /redfish/v1/TelemetryService                                 | `GET`          |
+| /redfish/v1/TelemetryService/MetricDefinitions               | `GET`          |
+| /redfish/v1/TelemetryService/MetricDefinitions/{MetricDefinitionId} | `GET`          |
+| /redfish/v1//TelemetryService/MetricReportDefinitions        | `GET`,         |
```
Comment (shivacharanms) [inline docs/README.md]: "," to be removed

File: docs/README.md [hunk 2]
```diff
@@ -568,6 +574,9 @@ Transfer-Encoding:chunked
    "Tasks": {
       "@odata.id": "/redfish/v1/TaskService"
    },
+   "Tasks": {
```
Comment (shivacharanms) [inline docs/README.md]: Should be TelemetryService

File: docs/README.md [hunk 3]
```diff
@@ -8563,30 +8706,10 @@ curl -i DELETE \
 ```
 curl -i -X DELETE \
    -H "X-Auth-Token:{X-Auth-Token}" \
- 'https://{odimra_host}:{port}/redfish/v1/Fabrics/{fabricID}/AddressPools/{addresspoolid}'
+ 'https://{odimra_host}:{port}/redfish/v1/Fabrics/{fabricID}/AddressPools/{addresspo'
```
Comment (shivacharanms) [inline docs/README.md]: Fix on format

File: docs/README.md [hunk 4]
```diff
@@ -10033,5 +10090,587 @@ curl -i GET \
...
+- Transmit metric reports with metric readings and any metadata associated with the readings
+- Specify trigger thresholds for a list of metrics
+
+**Supported APIs**
+
+| API URI                                                      | Operation Applicable | Required privileges     |
+| ------------------------------------------------------------ | -------------------- | ----------------------- |
+| /redfish/v1/TelemetryService                                 | GET                  | `Login`                 |
+| /redfish/v1/TelemetryService/MetricDefinitions               | GET                  | `Login`                 |
+| /redfish/v1/TelemetryService/MetricDefinitions/{MetricDefinitionID} | GET                  | `Login`                 |
```
Comment (shivacharanms) [inline docs/README.md]: Indent format fix

File: docs/README.md [hunk 5]
```diff
@@ -10033,5 +10090,587 @@ curl -i GET \
...
+              'https://{odimra_host}:{port}/redfish/v1/TelemetryService/MetricDefinitions/'
+
+```
+
+
+>**Sample response body**
+
+```
+{
+   "@odata.id":"/redfish/v1/TelemetryService/MetricDefinitions/PowerConsumedWatts",
```
Comment (shivacharanms) [inline docs/README.md]: Sample response to collection should be a collection, not individual resource info

File: docs/README.md [hunk 6]
```diff
@@ -10033,5 +10090,587 @@ curl -i GET \
...
+curl -i GET \
+              'https://{odimra_host}:{port}/redfish/v1/TelemetryService/MetricReportDefinitions/'
+
+```
+
+
+>**Sample response body**
+
+```
+{
```
Comment (shivacharanms) [inline docs/README.md:10299]: Response to collection should be collection of members

File: docs/README.md [hunk 7]
```diff
@@ -10033,5 +10090,587 @@ curl -i GET \
...
+              'https://{odimra_host}:{port}/redfish/v1/TelemetryService/Triggers/'
+
+```
+
+
+>**Sample response body**
+
+```
+{
+   "@odata.id":"/redfish/v1/TelemetryService/Triggers/PlatformPowerCapTriggers",
```
Comment (shivacharanms) [inline docs/README.md]: Response to collection should be collection of members

File: docs/README.md [hunk 8]
```diff
@@ -10033,5 +10090,587 @@ curl -i GET \
...
+
+```
+curl -i -X PATCH \
+   -H "Authorization:Basic YWRtaW46T2QhbTEyJDQ=" \
+   -H "Content-Type:application/json" \
+   -d \
+'{
+  "EventTriggers": ["Alert"]
+}' \
+ 'https://10.24.1.33:30080/redfish/v1/TelemetryService/Triggers/{TriggersID}'
```
Comment (shivacharanms) [inline docs/README.md]: Remove IP address

**Review by shivacharanms — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md]: "TelemetryService" is also added below just below "TaskService". Please verify.

File: docs/README.md [hunk 9]
```diff
@@ -10177,7 +10177,7 @@ curl -i GET \
 | **Method**         | `GET`                                                        |
 | ------------------ | ------------------------------------------------------------ |
 | **URI**            | `/redfish/v1/TelemetryService/MetricDefinitions`             |
-| **Description**    | This operation lists the metadata information for the metrics collection in Redfish implementation. |
+| **Description**    | This operation lists the metadata information for the metrics collection. |
```
Comment (shivacharanms) [inline docs/README.md]: change metrics to metric definitions

File: docs/README.md [hunk 10]
```diff
@@ -10234,7 +10228,7 @@ curl -i GET \
 | **Method**         | `GET`                                                        |
 | ------------------ | ------------------------------------------------------------ |
 | **URI**            | `/redfish/v1/TelemetryService/MetricDefinitions/{MetricDefinitionID}` |
-| **Description**    | This operation lists the metadata information for a metric in Redfish implementation. |
+| **Description**    | This operation lists the metadata information for a metric.  |
```
Comment (shivacharanms) [inline docs/README.md]: change metric to metric definition

File: docs/README.md [hunk 11]
```diff
@@ -10288,8 +10274,8 @@ curl -i GET \
 | **Method**         | `GET`                                                        |
 | ------------------ | ------------------------------------------------------------ |
 | **URI**            | `/redfish/v1/TelemetryService/MetricReportDefinitions`       |
-| **Description**    | This operation represents a set of metric properties of collected in multiple metric reports. |
-| **Returns**        | JSON schema defining the content and time of the metric reports |
+| **Description**    | This operation represents a collection of metric report definitions. |
+| **Returns**        | JSON schema defining the content of the multiple metric report |
```
Comment (shivacharanms) [inline docs/README.md]: change multiple to collection and metric report to metric report definition

File: docs/README.md [hunk 12]
```diff
@@ -10360,8 +10320,8 @@ MaxConsumedWatts"
 | **Method**         | `GET`                                                        |
 | ------------------ | ------------------------------------------------------------ |
 | **URI**            | `/redfish/v1/TelemetryService/MetricReportDefinitions/{MetricReportDefinitionID}` |
-| **Description**    | This operation represents metric properties of a single metric report. |
-| **Returns**        | JSON schema defining the content and periodicity of the metric report |
+| **Description**    | This operation represents a single metric report definition. |
+| **Returns**        | JSON schema defining the content of the single metric report |
```
Comment (shivacharanms) [inline docs/README.md]: change metric report to metric report definition

**Review by shivacharanms — CHANGES_REQUESTED** [review]
Please change the heading of this section from "Redfish Telemetry" to "Telemetry"

File: docs/README.md [hunk 13]
```diff
@@ -10033,5 +10090,530 @@ curl -i GET \
...
+| **Returns**        | JSON schema containing the definition, metadata or the characteristics of the metrics collection |
+| **Response Code**  | `200 OK`                                                     |
+| **Authentication** | No                                                           |
+
+
+>**curl command**
+
+```
+curl -i GET \
+              'https://{odimra_host}:{port}/redfish/v1/TelemetryService/MetricDefinitions/'
```
Comment (irappa-pattar) [inline docs/README.md:10192]: please remove trailing slash in the URI

File: docs/README.md [hunk 14]
```diff
@@ -10033,5 +10090,530 @@ curl -i GET \
...
+   "@odata.id":"/redfish/v1/TelemetryService/MetricReports/CPUUtilCustom1",
+   "@odata.type":"#MetricReport.v1_0_0.MetricReport",
+   "Id":"CPUUtilCustom1",
+   "Description":"Metric report of CPU Utilization for 10 minutes with sensing interval of 20 seconds.",
+   "MetricReportDefinition":{
+      "@odata.id":"/redfish/v1/TelemetryService/MetricReportDefinitions/CPUUtilCustom1"
+   },
+   "MetricValues":[
+      {
+         "MetricDefinition":{
```
Comment (irappa-pattar) [inline docs/README.md:10458]: MetricProperty Should be added in MetricValues.

**Review by irappa-pattar — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 14 above)
Comment (sunithasomanna) [reply docs/README.md:10458]: Addressed

File: docs/README.md (hunk 13 above)
Comment (sunithasomanna) [reply docs/README.md:10192]: Retaining this after confirming with Bharath B

**Review by irappa-pattar — APPROVED** [review]
LGTM

**Review by shivacharanms — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Pull request merged**